draft-ietf-rats-corim.md

```diff
@@ -1128,10 +1136,12 @@ the object relates to the subject.
 #### Device Identity Triple {#sec-comid-triple-identity}
 
 A Device Identity triple relates one or more cryptographic keys to a device.
-The subject of an Identity triple uses an instance or class identifier to refer
+The subject of an identity triple uses an instance or class identifier to refer
 to a device, and a cryptographic key is the object. The predicate asserts that
-the identity is authenticated by the key. A common application for this triple
-is device identity.
+at least one of the keys, as defined by `$crypto-key-type-choice`, authenticates the Attester.
```

Comment:
> at least one of the keys, as defined by `$crypto-key-type-choice`, authenticates the Attester

Should it be renamed to "Attester Identity Triple" then?

Reply:
Here device is the application layer concept that is been mapped to a `Environment` for encoding.

Perhaps, we can clarify better that the Device Identifier, either class of Device or specific Device instance, is expressed in the Environment in the subject of the triple

Reply:
@nedmsmith to wordsmith this